### PR TITLE
feat(passkey): allow for `attestationType: "direct"`

### PR DESCRIPTION
**Pull request by JuanM04**
Following what's described in #6371 (and old PR #2466), this PR allows for `attestationType: "direct"`. This implies always requesting AAGUID from the passkey provider, since with `"none"` it sometimes defaulted to `00000000-0000-0000-0000-000000000000`.

---
## Summary by cubic
Add support for `attestationType: "direct"` in passkey registration while keeping the default `"none"`. Added tests to ensure the default and forwarding behavior.

- **New Features**
  - Add `attestationType?: "none" | "direct"` to `PasskeyOptions`.
  - Forward `opts.attestationType` to registration; fallback to `"none"`.
  - Document `attestationType` in `docs/plugins/passkey.mdx` and add tests for default `"none"` and `"direct"`.

<sup>Written for commit bd57d0475aabf613b59de8ba22e44f9076c72f3c. Summary will update on new commits.</sup>

### CODE REVIEW

File: packages/passkey/src/routes.ts [hunk 1]
```diff
@@ -192,7 +192,7 @@ export const generatePasskeyRegistrationOptions = (
 				userID,
 				userName: ctx.query?.name || session.user.email || session.user.id,
 				userDisplayName: session.user.email || session.user.id,
-				attestationType: "none",
+				attestationType: opts.attestationType || "none",
 				excludeCredentials: userPasskeys.map((passkey) => ({
```
[the remark this answers is not in the document]
Comment (JuanM04) [reply packages/passkey/src/routes.ts:196]: Added test on last commit

**Comment by gustavovalverde** [comment]
Closing in favor of #9449. #9449 covers all four WebAuthn attestation values (`none`, `indirect`, `direct`, `enterprise`), renames the option to match the WebAuthn spec property (`attestation`), and includes a parameterized `it.each` test; this PR is limited to `none`/`direct` and has merge conflicts.
